scripts/check_if_pr_is_low_risk.py

```diff
@@ -0,0 +1,289 @@
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Check whether a PR is low-risk.
+
+A low-risk PR can be merged without running the full CI checks. When
+called with a URL to a PR, this script exits with code 0 if and only if
+the PR is low-risk. Otherwise, the exit code is nonzero.
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import argparse
+import json
+import re
+import sys
+
+import python_utils
+
+
+GITHUB_API_PR_ENDPOINT = (
+    'https://api.github.com/repos/%s/%s/pulls/%s')
+PR_URL_REGEX = (
+    r'^https://github.com/(?P<owner>\w+)/(?P<repo>\w+)/pull/(?P<num>\w+)/?$')
+
+
+def parse_pr_url(pr_url):
+    """Extract the owner, repo, and PR number from a PR URL.
+
+    For example, in the URL https://github.com/foobar/oppia/pull/23, the
+    owner is `foobar`, the repo is `oppia`, and the PR number is `23`.
+
+    Args:
+        pr_url: str. URL to the PR on GitHub.
+
+    Returns:
+        str, str, str. 3-tuple of the PR owner, repository, and number.
+    """
+    match = re.match(PR_URL_REGEX, pr_url)
+    if match:
+        return match.group('owner', 'repo', 'num')
+    else:
+        return None
+
+
+def load_diff(url):
```

Comment:
should we make the param pr_diff so that this function completely handles loading diff ii.e, so one can use it with random urls.

Reply:
I don't understand what you mean. This function downloads a PR diff from GitHub, so it cannot accept a PR diff as a parameter. If we already had a diff to pass into this function, we'd have no reason to call it in the first place

Reply:
Ah, sorry I mean pr_id* (not pr_diff*)

Reply:
this comment is no longer relevant since we don't use the github api for the diffs anymore